src/config.rs

```diff
@@ -14,6 +19,43 @@ pub struct Config {
     pub relay: Relay,
     pub prometheus: Prometheus,
     pub cors: bool,
+    pub keys: Keys,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, PartialEq, Eq)]
+#[serde(tag = "type")]
+pub enum Keys {
+    Static(Static),
+}
+
+impl Default for Keys {
+    fn default() -> Self {
+        Self::Static(Default::default())
+    }
+}
+
+#[serde_as]
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, PartialEq, Eq)]
+pub struct Static {
+    #[serde_as(as = "Base64<UrlSafe, Unpadded>")]
+    secret: Vec<u8>,
+}
+
+impl From<Static> for StaticSecret {
+    fn from(s: Static) -> Self {
+        StaticSecret::new(s.secret)
+    }
+}
+
+impl Default for Static {
+    fn default() -> Self {
+        Self {
+            // bit of a cludge, but we need to impl default :(
```

Comment:
Should it panic instead? What happens if the secret is lost, is the orbit basically "lost"?

Reply:
that would be acceptable, yeah if it's lost the host can't operate any orbits which used peerIds derived from the secret. maybe this should only be passable as an env var generally, same as the AWS credentials?